packages/prettier-plugin-java/test/test-utils.ts

```diff
@@ -1,18 +1,21 @@
 /*eslint no-console: ["error", { allow: ["error"] }] */
 
 import { expect } from "chai";
-import { readFileSync, existsSync, removeSync, copySync } from "fs-extra";
-import { resolve, relative, basename } from "path";
+import fs from "fs-extra";
+import { resolve, relative, basename, dirname } from "path";
 import klawSync from "klaw-sync";
 import { spawnSync } from "child_process";
 
-import { createPrettierDoc } from "../src/cst-printer";
+import { createPrettierDoc } from "../src/cst-printer.js";
 import { parse } from "java-parser";
 import { format, doc } from "prettier";
+import url from "url";
 
 const { printDocToString } = doc.printer;
+const { readFileSync, existsSync, removeSync, copySync } = fs;
```

Comment:
Correct. Because of the conversion to ESM, and because `fs-extra` is a CJS module, I got the following error:
```
import { readFileSync, existsSync, removeSync, copySync } from "fs-extra";
                                               ^^^^^^^^
SyntaxError: Named export 'copySync' not found. The requested module 'fs-extra' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'fs-extra';
const { readFileSync, existsSync, removeSync, copySync } = pkg;
```
I tried upgrading `fs-extra` to its latest version (11.1.1), but received the same error.

In the future we should be able to replace this usage of `fs-extra` with `fs`, since it contains equivalent functions to these 4, but for this PR I figured that would be inappropriate.

Reply:
Makes sense, we can create an issue to follow this one